src/goto-programs/lazy_goto_model.cpp

```diff
@@ -202,13 +202,37 @@ void lazy_goto_modelt::initialize(const cmdlinet &cmdline)
 /// Eagerly loads all functions from the symbol table.
 void lazy_goto_modelt::load_all_functions() const
 {
-  goto_convert(*goto_model, message_handler);
-}
+  std::vector<irep_idt> fn_ids_to_convert;
+  symbol_tablet::symbolst::size_type table_size;
+  symbol_tablet::symbolst::size_type new_table_size=symbol_table.symbols.size();
+  do
+  {
+    table_size=new_table_size;
+
+    // Find symbols that correspond to functions
+    fn_ids_to_convert.clear();
```

Comment:
Just use an object that's local to the loop body, please.

Reply:
This is the scoping I mentioned in my last comment. Hopefully that's all you meant.